### PR TITLE
FEA-134 Add loading bar on map when API call is performed

### PR DESCRIPTION
**Pull request by JakubLukaszCC**

### CODE REVIEW

File: src/pages/Map/components/MapLoadingBar/index.ts [hunk 1]
```diff
@@ -0,0 +1 @@
+export { MapLoadingBar } from './MapLoadingBar';
```
Comment (adrian-konopnicki-CC) [inline src/pages/Map/components/MapLoadingBar/index.ts:1]: **suggestion** I think that it would be nice if you export `MapLoadingBarTestIds`

Comment (JakubLukaszCC) [reply src/pages/Map/components/MapLoadingBar/index.ts:1]: good point :)

File: src/pages/Map/components/MapLoadingBar/MapLoadingBar.styled.ts [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+    rgba(255, 255, 255, 0.7) -0.24%,
+    rgba(255, 255, 255, 0.8) 49.88%,
+    rgba(255, 255, 255, 0.7) 100%
+  );
+  display: flex;
+  justify-content: center;
+  align-items: center;
+`;
+
+const Text = styled.p`
```
Comment (bednarskimichal) [inline src/pages/Map/components/MapLoadingBar/MapLoadingBar.styled.ts:21]: it looks like `font-family: 'Lato'` is not applied here

Comment (JakubLukaszCC) [reply src/pages/Map/components/MapLoadingBar/MapLoadingBar.styled.ts:21]: right, thanks

**Review by bednarskimichal — APPROVED** [review]

**Review by klaudia-blazyczek-cc — APPROVED** [review]

**Review by darya-karpovich-cc — APPROVED** [review]

**Review by lukasz-konatowski-cc — APPROVED** [review]

**Review by kamil-borecki-cc — APPROVED** [review]

**Review by adrian-konopnicki-CC — APPROVED** [review]

**Pull request merged**